pkg/config/dashboard/dashboard.go

```diff
@@ -81,27 +81,27 @@ func GetChaosDashboardEnv() (*ChaosDashboardConfig, error) {
 func ParsePersistTTLConfig(config *PersistTTLConfig) (*ttlcontroller.TTLConfig, error) {
 	syncPeriod, err := time.ParseDuration(config.SyncPeriod)
 	if err != nil {
-		return nil, errors.WithStack(err)
+		return nil, errors.Wrap(err, "parse configuration sync period")
 	}
 
 	event, err := time.ParseDuration(config.Event)
 	if err != nil {
-		return nil, errors.WithStack(err)
+		return nil, errors.Wrap(err, "parse configuration sync period")
```

Comment:
maybe use a different error message, like `parse configuration TTL for event`?

Or you prefer to use "parse configuration sync period" for all of them? 🙈

Reply:
Oh, sorry, I will fix it.

Reply:
Fixed!